test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua

```diff
@@ -0,0 +1,40 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends the notification after successful subscription to interior vehicle data with default moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is not subscribed to modules
+--
+-- Sequence:
+-- 1. GetInteriorVD(subscribe=true, module_1) is requested
+-- SDL does:
```

Comment:
```suggestion
-- SDL does:
send GetInteriorVD(subscribe = true, module_1) request to HMI
```
I propose to add a step. This comment relates to next scripts too.